jms/src/test/scala/akka/stream/alpakka/jms/scaladsl/JmsAckConnectorsSpec.scala

```diff
@@ -0,0 +1,371 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
```

Comment:
Should be `2018` now :)

and no start year because that is the year when the code was created.

Reply:
I'm no Lightbend lawyer. This is how it was (in the other files). I can change it as you wish provided it does not cause any other effect. Please confirm.

Reply:
Just leave it, the sbt header plugin will replace it with 2018 before we release anyway.

Reply:
That also means all files I've touched as part of this PR need to be updated to `2016-2018`. Making the changes now.

Reply:
Sorry, I've made all these changes. All it takes is one build to undo my changes. `The sbt-header` plugin is doing it and the configuration is here: https://github.com/akka/alpakka/blob/master/project/Common.scala#L53

That change should not be in scope for this PR.

Reply:
Sorry, @ennru. Somehow I did not see your comments during daytime today.

Reply:
I forgot that we (will) have the plugin release plugin, you can just ignore this.